samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -255,6 +255,18 @@ Testing Personalized Name extension is described in `Fast Pair Certification Gui
 .. note::
    To mitigate Android Personalized Name write issues, whenever you change the Personalized Name on an Android phone, you have to disconnect the phone from the Fast Pair Provider and wait for the phone to reconnect and write the new Personalized Name.
 
+Battery Notification extension
+------------------------------
+
+.. note::
+   Currently, Android phones have trouble with the Battery Notification extension and sometimes do not display battery information.
```

Comment:
Could we please move this note after line no: 269 so as to avoid a note just after the heading.